src/Pages/ChatRoom.tsx

```diff
@@ -0,0 +1,71 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState, useEffect } from 'react';
+import { css } from '@emotion/react';
+import ChatData from '../../public/data/chatting.json';
+import ChatHeader from '../components/ChatHeader';
+import Chatting from '../components/Chatting';
+import Input from '../components/Input';
+import { useChat } from '../hook/useChat';
+import { useFocus } from '../hook/useFocus';
+// import { useAlert } from '../hook/useAlert';
+
+function ChatRoom() {
+  const [chats] = useState(ChatData.chats);
```

Comment:
`setState`가 불필요하다면 state가 아닌 그냥 변수로 선언해도 좋을 것 같아요!
```suggestion
  const chats = ChatData.chats;
```
또는
```suggestion
  const { chats } = ChatData;
```